ga4gh/cli.py

```diff
@@ -34,173 +32,6 @@
 AVRO_LONG_MAX = 2**31 - 1
```

Comment:
This is a separate issue, but this should probably be moved to another file, for generality... like `avro_constants` or even in `protocol`